ariac-competitor/Dockerfile_xenial

```diff
@@ -0,0 +1,43 @@
+FROM osrf/ros:kinetic-desktop-full
+
+USER root
+
+ARG UBUNTU_DISTRO_TO_BUILD
+ARG ROS_DISTRO_BUILD_TIME
+
+# install bootstrap tools
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    python-rosdep \
+    python-rosinstall \
+    python-vcstools \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
+RUN wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
+RUN echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable ${UBUNTU_DISTRO_TO_BUILD} main" > /etc/apt/sources.list.d/gazebo-stable.list
+
+# Install the ariac package
+RUN apt-get update && apt-get install -y \
+        ros-${ROS_DISTRO_BUILD_TIME}-ros-core \  
+        ros-${ROS_DISTRO_BUILD_TIME}-ros-base \
+        ros-${ROS_DISTRO_BUILD_TIME}-catkin \
+        ros-${ROS_DISTRO_BUILD_TIME}-ros-controllers \
```

Comment:
All these packages are already provided by the `ros:kinetic-desktop-full` docker image so installing only the `ariac` package in this RUN command should be enough. (same below for Dockerfile_trusty)